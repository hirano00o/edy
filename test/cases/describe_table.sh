#!/bin/bash

SCRIPT_ROOT_DIR=$1
TEST_NAME=$(basename "$0" | sed "s/\..*//")

# aws dynamodb describe-table --table-name User --endpoint-url http://localhost:8000
CMD="edy d -t User --local 8000"

. "${SCRIPT_ROOT_DIR}"/helper.sh

run_such_query_helper