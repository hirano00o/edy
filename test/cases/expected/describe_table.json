{
  "tableArn": "arn:aws:dynamodb:ddblocal:000000000000:table/User",
  "tableName": "User",
  "partitionKey": {
    "name": "ID",
    "type": "N"
  },
  "sortKey": {
    "name": "Name",
    "type": "S"
  },
  "gsi": [
    {
      "indexName": "EmailIndex",
      "partitionKey": {
        "name": "Email",
        "type": "S"
      }
    }
  ],
  "itemCount": 8
}
